ui/app/src/Utils/Filter.elm

```diff
@@ -48,7 +48,7 @@ generateQueryString { receiver, showSilenced, text, group } =
     let
         -- TODO: Re-add receiver once it is parsed on the server side.
         parts =
-            [ ( "silenced", Maybe.map (toString >> String.toLower) showSilenced )
+            [ ( "silenced", Maybe.withDefault False showSilenced |> toString |> String.toLower |> Just )
```

Comment:
If it is `Nothing` we will not add the `silenced` query parameter to the API call. If we don't send the `silenced` parameter to the API, the back-end will send all alerts (silenced & !silenced).

When the user opens up Alertmanager without any parameters, `filter/showSilenced` will be `Nothing`. We want to hide silenced alerts by default, so the `generateQueryString` has to add the `silenced=false` query parameter in our case, where `showSilenced` is `Nothing`.

On the `/alerts` API endpoint returning all alerts should be the default.
On the `/alerts` UI page displaying only !silenced alerts should be the default.

Happy for any feedback or alternative solutions.